lua/lspconfig/server_configurations/sparql_ls.lua

```diff
@@ -0,0 +1,28 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'sparql-language-server'
+local full_path = util.find_bin_path(bin_name)
+
+local cmd = { 'node', full_path, '--stdio' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', 'node', full_path, '--stdio' }
```

Comment:
probably time for this to be lifted into util as well. That at least "stops the bleeding" and makes it easier to migrate to a full solution later.